### DISCUSSION TITLE
TFT  display  instead  Serial  for  AudioToolsLogger

### DISCUSSION THREAD

**Discussion by vednildip** [pschatzmann/arduino-audio-tools#1813] [Q&A] [answered]
Many  time  we  are  running  project  apart  from  PC , In  that  case  we  use  TFT  display  but  in  function  call  AudioToolsLogger.begin(Serial, AudioToolsLogLevel::Info);  instead  'Serial'  if  able  to  pass  ' TFT ' or  any  display  object  will  be  another  view  of  AudioToolsLogger.

**Comment by pschatzmann** [comment, marked as answer]
I don't quite understand: just use a TFT library that provides an output class that inherits from Print!
If you selected a library that does not do this, you can still just implement a simple adapter class yourself.
